config/giantswarm-vault-access/manager_init_container_patch.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
```

Comment:
what namespace is this?

Reply:
That's what you have `namespace`/`namePrefix`/`nameSuffix` in `kustomization.yaml` for. So you can keep the names generic and always set the namespace as `system`.

Reply:
Check the generated helm templates. They are all prefixed.